hermit/build.rs

```diff
@@ -177,12 +177,13 @@ impl KernelSrc {
 			self.src_dir.join("rust-toolchain.toml").display()
 		);
 
-		// HERMIT_LOG_LEVEL_FILTER sets the log level filter at compile time
-		println!("cargo:rerun-if-env-changed=HERMIT_MANIFEST_DIR");
-		println!("cargo:rerun-if-env-changed=HERMIT_LOG_LEVEL_FILTER");
 		println!("cargo:rerun-if-env-changed=HERMIT_CAREFUL");
-		println!("cargo:rerun-if-env-changed=HERMIT_MTU");
+		println!("cargo:rerun-if-env-changed=HERMIT_ENCODED_RUSTFLAGS");
+		println!("cargo:rerun-if-env-changed=HERMIT_LOG_LEVEL_FILTER");
+		println!("cargo:rerun-if-env-changed=HERMIT_MANIFEST_DIR");
 		println!("cargo:rerun-if-env-changed=HERMIT_MRG_RXBUF_SIZE");
+		println!("cargo:rerun-if-env-changed=HERMIT_MTU");
```

Comment:
It would be nice if this list of env vars could be taken from a file in the hermit kernel sources, such that it can't get out of sync as easily when a different kernel is chosen via `HERMIT_MANIFEST_DIR`.